src/backup.py

```diff
@@ -24,13 +24,15 @@ def configure(settings):
         f.write(config)
 
 
+def needs_setup():
+    return run("../backup/needs-setup.sh", stdout=DEVNULL, stderr=DEVNULL).returncode == 1
+
```

Comment:
I can't find the file `needs-setup.sh` in the submodule, is it missing a commit?